k8s/manifests/charts/ck-loadbalancer/values.yaml

```diff
@@ -1,3 +1,4 @@
+driver:
```

Comment:
I think we could set "cilium" as our default driver here to avoid leaving a value undefined.

Reply:
This has been brought up before. Conclusion is to not hide implicit defaults and let the implementation define it.